src/__tests__/tokenList.test.ts

```diff
@@ -2,12 +2,12 @@ import { expect } from "chai";
 
 import { fetchTokenList } from "../hooks/token";
 
-const configuredNetworks = ["MAINNET", "RINKEBY", "XDAI"];
+const configuredNetworks = [1, 4, 100];
```

Comment:
How did you find out those magic numbers? Should we probably have some constants for these?

Reply:
These are standard values for those networks. In the token list file I pus a constant call networkMap. We could import this and use it here.

Reply:
Also might be able to put this in a project config file. I'll try it out later.